docs/index.yml

```
@@ -202,53 +196,57 @@ additionalContent:
-          - url: architecture/microservices/index.md
-            text: ".NET Microservices: Architecture for Containerized .NET Applications"
-      # Card
       - title: Mobile
         links:
-          - url: /xamarin/ios/
+          - url: /xamarin/xamarin-forms
+            text: Xamarin.Forms
+          - url: /xamarin/ios
             text: Xamarin.iOS
           - url: /xamarin/android
             text: Xamarin.Android
-          - url: /xamarin/xamarin-forms
-            text: Xamarin.Forms
           - url: /azure/developer/mobile-apps
             text: Develop Xamarin apps with Azure
       # Card
       - title: Desktop
         links:
           - url: /uwp
             text: Universal Windows apps
-          - url: /dotnet/desktop/wpf/
-            text: Windows Presentation Foundation (.NET Core)
+          - url: /dotnet/desktop/wpf/?view=netdesktop-5.0
+            text: Windows Presentation Foundation (.NET 5)
           - url: /dotnet/desktop/wpf/?view=netframeworkdesktop-4.8
             text: Windows Presentation Foundation (.NET Framework)
+          - url: /dotnet/desktop/winforms/?view=netdesktop-5.0
+            text: Windows Forms (.NET 5)
           - url: /dotnet/desktop/winforms/?view=netframeworkdesktop-4.8
-            text: Windows Forms
+            text: Windows Forms (.NET Framework)
           - url: /xamarin/mac
             text: Xamarin for macOS
       # Card
-      - title: Gaming
+      - title: Microservices
         links:
-          - url: https://visualstudio.microsoft.com/vs/features/game-development/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link
-            text: Game development with Visual Studio
-          - url: https://docs.cryengine.com/display/CEPROG/C%23+Programming
-            text: Learn how to use CRYENGINE to build games with C#
-          - url: https://docs.monogame.net/?page=main
-            text: Build games with C# using the MonoGame library
-          - url: https://docs.unity3d.com/Manual/index.html
-            text: Learn how to use Unity to build 2D and 3D games with C#
+          - url: architecture/dapr-for-net-developers/index.md
+            text: Dapr for .NET Developers
+          - url: architecture/cloud-native/index.md
+            text: Cloud native .NET apps
+          - url: https://dotnet.microsoft.com/learn/aspnet/microservice-tutorial/intro
+            text: Create your first .NET microservice
+          - url: architecture/serverless/index.md
+            text: Serverless apps with Azure
+          - url: architecture/microservices/index.md
+            text: Architecture for Containerized .NET apps
+      # Card
+      - title: Cloud
+        links:
+          - url: azure/index.yml
+            text: Azure for .NET developers
+          - url: /dotnet/azure/migration/app-service?view=azure-dotnet
+            text: Migrate .NET web app or service
```

In its current state, I read the link text and ask myself "Migrate from where?" Should this text include the term "on-premises"?

```suggestion
            text: Migrate on-premises .NET web app or service
```